content/zh/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -415,30 +417,30 @@ of it.
 This admission controller mitigates the problem where the API server gets flooded by
 event requests. The cluster admin can specify event rate limits by:
 -->
-该准入控制器缓解了事件请求淹没 API 服务器的问题。集群管理员可以通过以下方式指定事件速率限制：
+此准入控制器缓解了事件请求淹没 API 服务器的问题。集群管理员可以通过以下方式指定事件速率限制：
 
 <!--
 * Enabling the `EventRateLimit` admission controller;
 * Referencing an `EventRateLimit` configuration file from the file provided to the API
   server's command line flag `--admission-control-config-file`:
 -->
 * 启用 `EventRateLimit` 准入控制器；
-* 从文件中引用 `EventRateLimit` 配置文件，并提供给 API 服务器命令的
-  `--admission-control-config-file` 标志：
+* 在通过 API 服务器的命令行标志 `--admission-control-config-file` 设置的文件中，
+  引用 `EventRateLimit` 配置文件，：
 
-```yaml
-apiVersion: apiserver.config.k8s.io/v1
-kind: AdmissionConfiguration
-plugins:
-- name: EventRateLimit
-  path: eventconfig.yaml
-...
-```
+  ```yaml
+  apiVersion: apiserver.config.k8s.io/v1
+  kind: AdmissionConfiguration
+  plugins:
+    - name: EventRateLimit
+      path: eventconfig.yaml
```

Comment:
```suggestion
  plugins:
  - name: EventRateLimit
    path: eventconfig.yaml
```

Reply:
These lines are indented for ease of reading.